### ISSUE TITLE
training and eval dataset setting

### ISSUE THREAD

**Issue by tsinghuarj** [google-research-datasets/dstc8-schema-guided-dialogue#25]
1、Rules 3：Participants are allowed to use any external datasets, resources or pre-trained models. 
2、the FILE_RANGES 
```json
{
    "dstc8_single_domain": {
        "train": range(1, 44),
        "dev": range(1, 8),
        "test": range(1, 8)
    },
    "dstc8_multi_domain": {
        "train": range(44, 128),
        "dev": range(8, 21),
        "test": range(8, 21)
    },
    "dstc8_all": {
        "train": range(1, 128),
        "dev": range(1, 21),
        "test": range(1, 21)
    }
}
```

Question:  can i use dstc8_all[train] (range(1, 128)) for training, and eval on dstc8_single_domain[test] or dstc8_multi_domain[test]?

**Comment by abhirast** [comment]
Yes, participants may use any of the splits for training. During test phase, we will release a set of json files containing unlabelled dialogues and participants are expected to submit the inference results on these dialogues. More details to follow next week.

**Comment by ypruan** [comment]
HI, so I'm wondering that whether participants can use dev set for model training?

**Comment by abhirast** [comment]
Hi. Sure, participants may use the dev set in any way they see fit. Though, please note that dev set has many services which are not present in the training set. A naive use of dev set in training will not preserve this setting and your model may do worse on the test set. Also, in case the dev set is used for training, we require participants to report both results - with and without training on the dev set. This is to facilitate comparison with submissions which didn't use dev set for training.
